deploy/docker/notebook/base/base.yaml

```diff
@@ -425,30 +426,36 @@ addfiles:
           if "NB_UMASK" in os.environ:
               os.umask(int(os.environ["NB_UMASK"], 8))
 
-          # Configure file post-save, post-rename, and post-delete hooks
-          
-          #### Hook Class ####
-
           class HookFileContentsManager(FileContentsManager):
               """ Jupyter Server Hook Class for database synchronization with Notebooks Hub
+              Note: os_path, os_old_path, and os_new_path only track relative file paths instead of absolute 
+              So we have to pre-pend the full path resolution with Path().resolve()
               """
 
               POLUS_NOTEBOOKS_HUB_API = os.environ["POLUS_NOTEBOOKS_HUB_API"]
-              work_path = '/home/jovyan/work/'
-              shared_path = '/home/jovyan/shared/notebooks'
+              DELETE_URI = urljoin(POLUS_NOTEBOOKS_HUB_API, '/file-data/delete-file')
+              RENAME_URI = urljoin(POLUS_NOTEBOOKS_HUB_API, '/file-data/rename-file')
+              SAVE_URI = urljoin(POLUS_NOTEBOOKS_HUB_API, '/file-data/new-file')
```

Comment:
Variable names should be lowercase following PEP-8 standard https://www.python.org/dev/peps/pep-0008/#function-and-variable-names